test/helpers/data/aggregated-data-helper.js

```diff
@@ -278,11 +278,20 @@ module.exports.selectIdsForWorkloadOwner = function () {
   return promise
 }
 
-module.exports.getAnyExistingTeamId = function () {
-  var promise = knex('team')
+module.exports.getAnyExistingWorkloadOwnerId = function () {
+  var promise = knex('workload_owner')
+      .first('id')
+      .then(function (result) {
+        return result.id
+      })
+  return promise
```

Comment:
Just a question, how come we return the promise here but elsewhere we usually just return the knex statement.  Just  style thing?

Reply:
Its just the style for the data helper.